### PR TITLE
Pull request on branch: develop

### PR DESCRIPTION
**Pull request by mutoromike**
Changes made to UI files and Ireframes

### CODE REVIEW

File: Designs/UI/css/styling.css [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+
+#nav > ul > a {
+	padding-right: 8px;
+}
+
+.clear {
+	clear: both;
+}
+#content {
+	width: 100%;
```
Comment (FelixWambiri) [inline Designs/UI/css/styling.css:163]: You can move this styling to be on the same level with the rest for consistency

File: Designs/UI/css/styling.css [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+    position: absolute;
+    left: 0;
+    bottom: 0;
+    height: 50px;
+    width: 100%;
+    overflow:hidden;
+}
+
+#footer-wrapper {
+	width: 100%;
```
Comment (FelixWambiri) [inline Designs/UI/css/styling.css:196]: Try to align all statements to be on same level for consistency

**Review by FelixWambiri — COMMENTED** [review]
Your work looks good and you have a very attractive UI.
I have commented on some parts in the code for consistency

File: Designs/UI/css/styling.css (hunk 2 above)
Comment (mutoromike) [reply Designs/UI/css/styling.css:196]: Thanks, will look into that Sir. Anything that I can add to the UI for improvement? Designwise...

**Pull request merged**